relevanceai/operations_new/ops.py

```diff
@@ -515,3 +515,99 @@ def apply_transformers_pipeline(
             output_field=output_field,
         )
         return ops.run(self, filters=filters, select_fields=text_fields)
+
+    def subcluster(
+        self,
+        vector_fields: List[str],
+        alias: str,
+        parent_field: str,
+        model: Any = "kmeans",
+        cluster_field: str = "_cluster_",
+        model_kwargs: Optional[dict] = None,
+        filters: Optional[list] = None,
+        cluster_ids: Optional[list] = None,
+        min_parent_cluster_size: int = 0,
+        **kwargs,
+    ):
+        from relevanceai.operations_new.cluster.sub.ops import SubClusterOps
+
+        ops = SubClusterOps(
+            model=model,
+            alias=alias,
+            vector_fields=vector_fields,
+            parent_field=parent_field,
+            model_kwargs=model_kwargs,
+            cluster_field=cluster_field,
+            credentials=self.credentials,
+            dataset_id=self.dataset_id,
+            cluster_ids=cluster_ids,
+            min_parent_cluster_size=min_parent_cluster_size,
+            **kwargs,
+        )
+
+        # Building an infinitely hackable SDK
+
+        # Add filters and select fields
+        select_fields = vector_fields + [parent_field]
+        if filters is None:
+            filters = []
+
+        if cluster_ids is not None:
+            filters += [
```

Comment:
possible to use `self[parent_field] == cluster_id` here?

Reply:
Yeah but using default filters just makes it slightly faster as it skips a layer of abstraction